parachain/runtime/interlay/src/lib.rs

```diff
@@ -1266,6 +1284,8 @@ construct_runtime! {
         DexGeneral: dex_general::{Pallet, Call, Storage, Event<T>} = 101,
         DexStable: dex_stable::{Pallet, Call, Storage, Event<T>}  = 102,
         DexSwapRouter: dex_swap_router::{Pallet, Call, Event<T>} = 103,
+
+        Recovery: pallet_recovery::{Pallet, Call, Storage, Event<T>} = 104,
```

Comment:
I don't think we should group this with the DeFi pallets, maybe index 11 is most suitable (after `TxPause`)?